Gayle-gahee-gayle/Homework-4/madlib-with-loops/js/madlib-console.js

```diff
@@ -0,0 +1,32 @@
+
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+var random1 = Math.floor((Math.random() * startupX.length));
+var random2 = Math.floor((Math.random() * startupY.length));
+
+console.log('A startup that is ' + startupX[random1] + ', but for ' + startupY[random2]);
+
+let $buttonCreate = $('#create');
+let $buttonSave = $('#save');
+let $buttonPrint = $('#print');
+let $outputH1 = $('#xForY');
+
+let $favorites = $('#favorites');
+let arrayFavorites = [];
+
+$buttonCreate.on('click', function(){
+  var random1 = Math.floor((Math.random() * startupX.length));
+  var random2 = Math.floor((Math.random() * startupY.length));
+  var newStartup = 'A startup that is ' + startupX[random1] + ', but for ' + startupY[random2];
+  $('#xForY').text(newStartup);
+
+  $buttonSave.on("click", function() {
+      arrayFavorites.push(newStartup);
```

Comment:
Gayle, you've come a long way in building a foundation in programming and it's clear your ability to quickly learn new concepts will take you far in building intuitive user experience designs in the future. 

The ability to communicate between designer and programmer is an important skill, and you've made significant strides to develop it. Keep it up! 👍